kytos/core/logs.py

```diff
@@ -115,6 +116,26 @@ def filter_session_disconnected(record):
                     and record.args and isinstance(record.args[0], str)
                     and record.args[0].endswith(msg_end))
 
+    @staticmethod
+    def decorate_logger_class(*decorators):
```

Comment:
@Ktmi Could you add unit tests for this staticmethod and the decorators that have been introduced?

Reply:
Added some unit tests for `decorate_logger_class`, `root_decorator`, and 'queue_decorator'. Wasn't able to figure a way to properly test `apm_decorator`.

Reply:
@Ktmi, yes, it's a bit tough to test cleanly this one. I've managed to cover it, feel free to cherry-pick this commit https://github.com/kytos-ng/kytos/commit/9d787bad927456037e8f4f468723a1b1cac267a0 or adapt it.

Reply:
Was able to figure out a solution to make sure that the apm methods are working as intended. I use mocks to make sure the transaction is properly processed with the appropriate begin and end span, while still calling the base method.